apiary.apib

```diff
@@ -3281,6 +3281,312 @@ Update the current timer. Only possible if there is a timer running.
+                + created_at: `2019-05-09T11:25:11+00:00` (string)
+                + updated_at: `2019-05-09T11:30:58+00:00` (string)
+
+### workOrders.info [GET /workOrders.info]
+
+Get details for a single work order.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string, required)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string)
+            + state: `draft` (enum[string])
+                + Members
+                    + draft
+                    + finalized
+                    + sent
+            + customer (object)
+                + type (enum[string])
+                    + Members
+                        + contact
+                        + company
+                + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string)
+            + starts_at: `2017-05-26T10:01:49+00:00` (string)
+            + description: `Pipework intervention` (string)
+            + department (object)
+                + type: `department` (string)
+                + id: `5e90eb0a-b502-4344-aa0f-3b8525af6186` (string)
+            + layout (object)
+                + type: `layout` (string)
+                + id: `a05320f6-b3ef-4ff9-a287-2074a12fb3e9` (string)
+            + milestone (object)
+                + type: `milestone` (string)
+                + id: `e94c1363-3426-46a4-921d-fce03f39bbd5` (string)
+            + event (object)
+                + type: `event` (string)
+                + id: `9fc14045-5c6c-4ba8-8672-42ea3f26aa63` (string)
+            + location (Address)
+            + distance (Distance)
+            + created_at: `2019-05-09T11:25:11+00:00` (string)
+            + updated_at: `2019-05-09T11:30:58+00:00` (string)
+            + time (array)
+                + (object)
+                    + user (object)
+                        + type: `user` (string)
+                        + id: `4ec55a8c-2d80-472a-a9c2-5ff5ee7eac6a` (string)
+                    + started_at: `2017-04-26T10:01:49+00:00` (string)
+                    + duration: 3600 (number) - In seconds
+                    + description (string)
+            + materials (array)
+                + (object)
+                    + product (object)
+                        + type: `product` (string)
+                        + id: `e2314517-3cab-4aa9-8471-450e73449040` (string)
+                    + quantity: 3 (number)
+            + images (array) - Only included with request parameter `includes=images`.
+                + (object)
+                    + id: `a856642a-a476-4b69-8e42-e7d1badb2ce9` (string)
+                    + location: `https://cdn.teamleader.eu/file` (string) - A temporary URL where the image can be downloaded
+                    + expires: `2018-02-05T16:44:33+00:00` (string) - Expiration time of the temporary download URL
```

Comment:
PP: Add a expires after the `created_at` of the WorkOrder.

Reply:
What do you mean here? A workOrder does not expire. It's the image link that expires.

Reply:
Not clear, sorry. 
The date on the `expires: 2018-02-05T16:44:33+00:00` is before the date the WorkOrder was created `created_at: 2019-05-09T11:25:11+00:00`. Should be after no? 
But, again, just a PP.

Reply:
Aha! I will make sure this is fixed in the actual implementation.
This PR will never get merged.